src/main/scala/org/camunda/feel/impl/builtin/StringBuiltinFunctions.scala

```diff
@@ -262,4 +264,13 @@ object StringBuiltinFunctions {
         ValString(string.trim)
       }
     )
+
+  private def uuidFunction =
+    builtinFunction(
+      params = List(),
+      invoke = { case List() =>
+        val uuid  = Generators.timeBasedGenerator.generate()
```

Comment:
:wrench: In Camunda 7, we create only one instance of the generator, by including the ethernet address ([ref](https://github.com/camunda/camunda-bpm-platform/blob/master/engine/src/main/java/org/camunda/bpm/engine/impl/persistence/StrongUuidGenerator.java#L44C32-L44C94)). We could do the same and define the generate as lazy val field.

```
private lazy val generator = Generators.timeBasedGenerator(EthernetAddress.fromInterface())

// ...
invoke = { case List() => ValString(generator.generate.toString()) }
```

Reply:
@saig0, It's done.